docs/docs/rest-api/public/api/v2/types/container-status.raml

```diff
@@ -0,0 +1,56 @@
+#%RAML 1.0 DataType
+type: object
+additionalProperties: false
+properties:
+  name:
+    type: !include name.raml
+    description: name of the container specification (within the pod)
+  status:
+    type: string
+    enum: [ STAGING, STABLE, DEGRADED, TERMINAL ]
+    description: |
+      STAGING  - Container has been launched but is not yet running.
+      STABLE   - Container is running and, if health checks were specified, is healthy.
+      DEGRADED - Container is running but its health check is not passing.
```

Comment:
maybe we should add a "reasonCode" or something. So that we can mark a
container as DEGRADED with reasonCode=unreachable?

On Fri, Aug 26, 2016 at 10:30 AM, Matthias Eichstedt <
notifications@github.com> wrote:

> In docs/docs/rest-api/public/api/v2/types/container-status.raml
> <https://github.com/mesosphere/marathon/pull/4250#discussion_r76427847>:
>
> > @@ -0,0 +1,56 @@
> > +#%RAML 1.0 DataType
> > +type: object
> > +additionalProperties: false
> > +properties:
> > +  name:
> > +    type: !include name.raml
> > +    description: name of the container specification (within the pod)
> > +  status:
> > +    type: string
> > +    enum: [ STAGING, STABLE, DEGRADED, TERMINAL ]
> > +    description: |
> > +      STAGING  - Container has been launched but is not yet running.
> > +      STABLE   - Container is running and, if health checks were specified, is healthy.
> > +      DEGRADED - Container is running but its health check is not passing.
>
> What would the status be if the tasks of a podInstance are Unreachable
> (LOST for now)? I think we need an UNREACHABLE here as well.
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/mesosphere/marathon/pull/4250/files/c1e350f76ab698dd26c9bfba42fb6d14995bc7b5#r76427847>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ACPVLCQLHsBnJliYdUiDwTVVGqpHPNklks5qjviCgaJpZM4Jsi1k>
> .
>